apps/prairielearn/python/python_helper_sympy.py

```diff
@@ -529,20 +667,16 @@ def validate_string_as_sympy(
     return None
 
 
-def get_variables_list(variables_string: Optional[str]) -> list[str]:
-    if variables_string is None:
+def get_items_list(items_string: Optional[str]) -> list[str]:
+    if items_string is None:
         return []
 
-    return list(map(str.strip, variables_string.split(",")))
+    return list(map(str.strip, items_string.split(",")))
 
 
 def process_student_input(student_input: str) -> str:
```

Comment:
Note to self: Remove this function and have this be part of the evaluation function.